src/build.cc

```diff
@@ -164,10 +164,12 @@ void BuildStatus::BuildFinished() {
 }
 
 string BuildStatus::FormatProgressStatus(
-    const char* progress_status_format) const {
+    const char* progress_status_format,
+    const string to_print) const {
```

Comment:
Passing to_print by value here makes a copy.  It's surely harmless but from a style perspective you should use `const string&` like elsewhere in Ninja.